src/WooCommerceCheckout.php

```diff
@@ -15,22 +15,26 @@ public static function init() {
     // Removes city field from shipping calculator.
     add_filter('woocommerce_shipping_calculator_enable_city', '__return_false');
 
-    if (is_admin()) {
+    if (is_admin() || wp_doing_cron()) {
```

Comment:
This file is only related to the WooCommerce Checkout so we can just return if we are doing a cron job.

Reply:
I understand the thinking but I would not advise to do this, because if we are going to add some checkout related customization in the future, then we will debug a long time why the system behaves differently during cron runs.
```suggestion
    if (is_admin()) {
```

Reply:
Makes sense. Thanks for pointing it out.